flang/include/flang/Parser/dump-parse-tree.h

```diff
@@ -693,6 +700,7 @@ class ParseTreeDumper {
   NODE(parser, OpenMPCriticalConstruct)
   NODE(parser, OpenMPDeclarativeAllocate)
   NODE(parser, OpenMPDeclarativeConstruct)
+  NODE(parser, OmpDeclareVariantDirective)
```

Comment:
Nit: Is there a reason this directive is called "Omp" and the others are "OpenMP"?

Reply:
In the standard it is listed as a directive.